packages/skeleton-svelte/src/lib/components/Listbox/ListboxItem.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+	import { createId } from '$lib/internal/create-id.js';
+	import { getListboxContext } from './context.js';
+	import { getFirstElement, getLastElement, getNextElement, getPreviousElement } from '$lib/internal/elements.js';
+	import type { ListboxItemProps } from './types.js';
+
+	// Props
+	let {
+		value,
+		base = 'flex items-center gap-2 text-left',
+		padding = 'py-2 px-4',
+		radii = 'rounded',
+		selected = 'preset-filled',
+		classes,
+		leadBase,
+		leadClasses,
+		trailBase,
+		trailClasses,
+		children,
+		lead,
+		trail,
+		...attributes
+	}: ListboxItemProps = $props();
+
+	// Context
+	const context = getListboxContext();
```

Comment:
I've used `ctx` in favor of `context` in other components. It's a common pattern I picked up from working with Canvas elements. We should converge on a standard here.

Reply:
Done, agreed.